src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -262,8 +267,8 @@
     <!-- FAQ -->
     <div class="faq">
       <!-- Title -->
-      <p class="grey-header">FAQ</p>
-      <h2>Frequently asked questions</h2>
+      <p class="grey-header vads-u-font-size--sm vads-u-margin-bottom--0">FAQ</p>
```

Comment:
Can we add `<abbr title="Frequently asked questions">FAQ</abbr>` here so screen readers don't try and pronounce this?